<launch>
    <arg name="leader_robot_name" value="robot_0"/>
	<arg name="slave_robot_1" value="robot_1"/>
    <arg name="slave_robot_2" value="robot_2"/>

	<arg name="slave1_x" value="-0.8"/> 
	<arg name="slave1_y" value="0.8"/>

    <arg name="slave2_x" value="-0.8"/>
    <arg name="slave2_y" value="-0.8"/>
    <!-- publish slave(s) expected location -->
    <node pkg="turtlebot_sim" type="switch_formation.py" name="switch_formation" output="screen">
        <param name="leader_robot_name" value="$(arg leader_robot_name)" />
        <param name="slave_robot_1" value="$(arg slave_robot_1)" />
        <param name="slave_robot_2" value="$(arg slave_robot_2)" />
        <param name="slave1_x" type="double" value="$(arg slave1_x)" />
        <param name="slave1_y" type="double" value="$(arg slave1_y)" />
        <param name="slave2_x" type="double" value="$(arg slave2_x)" />
        <param name="slave2_y" type="double" value="$(arg slave2_y)" />
    </node>
</launch>